src/test/java/api/support/builders/NoticePolicyBuilder.java

```diff
@@ -77,6 +94,7 @@ public JsonObject create() {
     put(request, "description", this.description);
     put(request, "active", this.active);
     put(request, "loanNotices", new JsonArray(loanNotices));
+    put(request, "requestNotices", new JsonArray(requestNotices));
```

Comment:
Does this property exist in the version of the interface currently declared as a dependency?

Reply:
Yes, it does.